code/datums/mind.dm

```diff
@@ -1069,8 +1069,14 @@ datum/mind
 		else if (href_list["common"])
 			switch(href_list["common"])
 				if("undress")
-					for(var/obj/item/W in current)
-						current.unEquip(W, 1)
+					if(istype(current, /mob/living/carbon/human))

```

Comment:
Huh. I explicitly remembered to use ishuman in one spot, and then forgot to in two.